pkg/core/native/policy.go

```diff
@@ -332,6 +348,41 @@ func (p *Policy) IsBlockedInternal(dao dao.DAO, hash util.Uint160) bool {
 	return dao.GetStorageItem(p.ContractID, key) != nil
 }
 
+func (p *Policy) getStoragePrice(ic *interop.Context, _ []stackitem.Item) stackitem.Item {
+	return stackitem.NewBigInteger(big.NewInt(p.GetStoragePriceInternal(ic.DAO)))
+}
+
+// GetStoragePriceInternal returns current execution fee factor.
+func (p *Policy) GetStoragePriceInternal(d dao.DAO) int64 {
```

Comment:
Let's return `uint32` from this method, beecause `GetStoragePriceInternal` is available from the outside and storage price cannot be negative.

Reply:
The same thing can be done for `ExecFeeFactor`.

Reply:
This is GAS which is `int64` both in P2P and other methods of this contract, so I think it's better to stick with it.